packages/expo-analytics-amplitude/src/ExpoAmplitude.web.ts

```diff
@@ -1,5 +1,31 @@
+import amplitude from 'amplitude-js'
+
 export default {
   get name(): string {
     return 'ExpoAmplitude';
   },
-};
+  initialize(apiKey: string, options?: amplitude.Config) {
+    amplitude.getInstance().init(apiKey, undefined, options)
```

Comment:
```suggestion
  initialize(apiKey: string) {
    amplitude.getInstance().init(apiKey);
```
As per third code block at https://developers.amplitude.com/docs/javascript#installing-with-npm-or-yarn

Reply:
@sjchmiela I based this on the info about `init` here: https://developers.amplitude.com/docs/javascript-sdk-reference (seems like the docs have changed and there is a broken link) but there are options here for web that don't exist in mobile that might be useful for people.